src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
+  const newSalary = document.createElement('td');
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.setAttribute('class', 'notification');
+
+  div.style.position = 'absolute';
+
+  const valid = checkValidValueError();
+
+  if (valid) {
+    createMessageError(div);
+  } else {
+    createMessagSaccess(div);
+  }
+
+  document.body.append(div);
+
+  if (div.getAttribute('data-qa') === 'error') {
+    div.style.background = 'rgb(153, 41, 26)';
+
+    setTimeout(() => {
+      div.style.display = 'none';
+    }, 4000);
+  }
+
+  newName.textContent = inputName.value;
+  newPosition.textContent = inputPosition.value;
+  newOffice.textContent = selectOffice.value;
+  newAge.textContent = inputAge.value;
+  newSalary.textContent = `$${(+inputSalary.value).toLocaleString('en-US')}`;
+
+  newRows.append(newName, newPosition, newOffice, newAge, newSalary);
+
+  setTimeout(() => {
+    div.style.display = 'none';
+  }, 4000);
+
+  const validSaccess = checkValidValueSaccess();
+
+  if (validSaccess) {
+    tbody.append(newRows);
+    form.reset();
+  }
+});
+
+title.addEventListener('click', (e) => {
+  const sortTody = document.querySelector('tbody');
+  const rows = [...sortTody.querySelectorAll('tr')];
+  const value = e.target.closest('th').textContent.toLowerCase();
+
+  const intex = e.target.cellIndex;
+  const cells = [];
+
+  sortTody.querySelectorAll('tr').forEach((item) => {
+    cells.push(item.cells[intex].textContent);
+  });
+
+  const newRows = rows.sort((a, b) => {
+    const cellsA = a.cells[intex].textContent;
+    const cellsB = b.cells[intex].textContent;
+
+    switch (value) {
+      case 'name':
+      case 'position':
+        return cellsA.localeCompare(cellsB);
+
+      case 'age':
+        return cellsA - cellsB;
+
+      case 'salary':
+        return (
+          +cellsA.replace(',', '').slice(1) - +cellsB.replace(',', '').slice(1)
+        );
+
+      default:
+        return null;
+    }
+  });
+
+  if (value === revers) {
+    newRows.reverse();
+    revers = '';
+  } else {
+    revers = value;
+  }
+
+  sortTody.innerHTML = '';
+
+  newRows.forEach((item) => {
+    sortTody.appendChild(item);
+  });
+});
+
+tbody.addEventListener('dblclick', (e) => {
+  const findInput = document.querySelector('.cell-input');
+  const oldValue = e.target.textContent;
+
+  if (findInput) {
+    findInput.remove();
+    e.target.textContent = oldValue;
+  }
+
+  const input = document.createElement('input');
+
+  input.setAttribute('class', 'cell-input');
+  e.target.textContent = '';
+  e.target.append(input);
+  input.focus();
+
+  if (
+    e.target === e.target.parentElement.children[3] ||
+    e.target === e.target.parentElement.children[4]
+  ) {
+    input.style.width = '60px';
+  } else {
+    input.style.width = '100px';
+  }
+
+  input.onkeydown = (even) => {
+    if (even.key === 'Enter') {
+      if (e.target === e.target.parentElement.children[3]) {
```

Comment:
You're setting the 'type' attribute of 'input' to 'number' inside the 'keydown' event. It should be set when the input is created, not during the event.